.github/workflows/build.yml

```diff
@@ -0,0 +1,73 @@
+name: build
+
+on:
+  workflow_dispatch:
+  workflow_call:
+    inputs:
+      os:
+        required: true
+        type: string
+      node:
+        required: false
+        type: string
+        default: ${{ vars.DEFAULT_NODE_VERSION }}
+      CACHE_DIR:
+        required: false
+        type: string
+        default: dist
+      NPM_NETWORK_TIMEOUT:
```

Comment:
Should we keep inputs lowercase for consistency?

Reply:
Can do.